Dockerfile

```diff
@@ -4,13 +4,13 @@ WORKDIR /srv/cartography
 
 ENV PATH=/venv/bin:$PATH
 RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends python3.8-dev python3-pip python3-setuptools openssl libssl-dev gcc pkg-config libffi-dev libxml2-dev libxmlsec1-dev curl && \
```

Comment:
Let's revert back all the changes in this file. We should not change `Dockerfile`. We should change only Cloudanix specific files.